web/pdf_viewer.css

```diff
@@ -65,6 +108,22 @@
   background: url('images/loading-icon.gif') center no-repeat;
 }
 
+.pdfPresentationMode .pdfViewer {
+  display: block;
```

Comment:
There is no new code that overrides `display`. Is this a redundant remnant from your Flexbox experiment? If so, remove it.